fusor-ember-cli/tests/acceptance/deployments-test.js

```diff
@@ -24,8 +24,8 @@ test('visiting /deployments', function(assert) {
 test('user should see all elements on deployments page', function(assert) {
 
     var org = server.create('organization');
-    var env = server.create('lifecycle_environment');
-    server.createList('deployment', 10, {organization_id: org.id, lifecycle_environment_id: env.id});
+    var env = server.create('lifecycle-environment');
+    server.createList('deployment', 10, {organization: org, lifecycleEnvironment: env});
```

Comment:
new convention for Mirage 0.2.x is to use dashes for model and relationship name rather than _id